ext/node/polyfills/http2.ts

```diff
@@ -210,11 +210,12 @@ export class Http2Session extends EventEmitter {
   }
 
   goaway(
-    _code: number,
-    _lastStreamID: number,
-    _opaqueData: Buffer | TypedArray | DataView,
+    code?: number,
+    lastStreamID?: number,
+    opaqueData?: Buffer | TypedArray | DataView,
   ) {
-    warnNotImplemented("Http2Session.goaway");
+    // TODO(satyarohith): create goaway op and pass the args
+    debugHttp2(">>> goaway - ignored args", code, lastStreamID, opaqueData);
```

Comment:
This is gonna be hard - last time I tried it, `h2`/`hyper` didn't allow to explicitly send `goaway` frame :(

Reply:
Yeah. I found that frame module is public under `unstable` feature flag but I need to check whether send_data() on SendStream allows us to send single frames (it does allow sending data frames but not sure about goaway).